dipy/reconst/dti.py

```diff
@@ -1132,6 +1132,9 @@ def predict(self, gtab, S0=1, step=None):
            The mean non-diffusion weighted signal in each voxel. Default: 1 in
            all voxels.
 
+        step : int
+            The chunk size as a number of voxels.
```

Comment:
Should the explanation come directly under the definition of step or I should include it under Notes. Also, the line - the increase in step value "should" speed things up but it ~~should~~ will also take up more memory - Is this correct? Regarding the memory 
```python
    """
            The chunk size as a number of voxels. Optional parameter with default value 10,000.

            In order to increase speed of processing, tensor fitting is done simultaneously
            over many voxels. This parameter sets the number of voxels that will be fit at 
            once in each iteration. A larger step value should speed things up, but will 
            also take up more memory. It is advisable to keep an eye on memory consumption 
            as this value is increased.
    """
```